DecipherNow/gm-fabric-dashboard
# chore: Update Node packages for docker prod image

Possibly resolves some security vulnerabilities around RegExp exploits flagged by Code Climate. See https://codeclimate.com/github/DecipherNow/gm-fabric-dashboard/issues?category=security

## Proposed Changes

  - Update packages used by our production Docker image
  
## How to Test

  - `npm run mock-sds`
  - `npm run build`
  - `npm run build-docker`
  - `docker run -it -p 1338:1337 deciphernow/gm-fabric-dashboard`
  - Open localhost:1338 in a browser and confirm that the app loads as expected.
